osu.Server.Spectator/Database/DatabaseAccess.cs

```diff
@@ -189,13 +180,34 @@ public async Task RemoveRoomParticipantAsync(MultiplayerRoom room, MultiplayerRo
         public async Task<long> AddPlaylistItemAsync(multiplayer_playlist_item item)
         {
             await connection.ExecuteAsync(
-                "INSERT INTO multiplayer_playlist_items (room_id, beatmap_id, ruleset_id, allowed_mods, required_mods, created_at, updated_at)"
-                + " VALUES (@room_id, @beatmap_id, @ruleset_id, @allowed_mods, @required_mods, NOW(), NOW())",
+                @"INSERT INTO multiplayer_playlist_items (room_id, beatmap_id, ruleset_id, allowed_mods, required_mods, created_at, updated_at)
+                VALUES (@room_id, @beatmap_id, @ruleset_id, @allowed_mods, @required_mods, NOW(), NOW())",
                 item);
 
             return await connection.QuerySingleAsync<long>("SELECT max(id) FROM multiplayer_playlist_items WHERE room_id = @room_id", item);
         }
 
+        public async Task UpdatePlaylistItemAsync(multiplayer_playlist_item item)
+        {
+            await connection.ExecuteAsync(
+                @"UPDATE multiplayer_playlist_items SET
```

Comment:
Try to avoid writing database queries with whitespace in the string, it makes viewing in mysql's processlist or potentially creating query rules for it quite hard. See original code which uses `+ ""` for each line instead.

Reply:
I'll take note of that, because I've done the same in other projects (e.g. https://github.com/ppy/osu-queue-score-statistics/pull/6), updated everything relevant in this file.